### ISSUE TITLE
Multiple LASSO benchmarks

### ISSUE THREAD

**Issue by tanglef** [benchopt/results#18]
There are currently two locations with LASSO outputs: `benchmark_lasso` and `lasso`, each displaying separately on the website.
And in the `lasso` one file is now following the naming convention thus not displayed online.
Maybe we should find a way to harmonize the naming with published results.

**Comment by agramfort** [comment]
yes the one called lasso is spurious. This folder should be removed

>

**Comment by tomMoral** [comment]
I think this should be independent from the name of the benchmark...
I kind of lazily decided to check the repo name but a cleaner way would be to set the benchmark name somewhere like in the objective?

**Comment by tomMoral** [comment]
This should be solved with the newest version of benchopt if we update the benchmark to put the `url` attribute in `Objective`.
